src/modules/assessments/components/AssessmentTitle.tsx

```diff
@@ -81,7 +89,17 @@ const AssessmentTitle = ({
           {assessmentTitle}
           {': '}
         </b>
-        {projectName}
+        <RouterLink
+          to={generateSafePath(Routes.ENROLLMENT_DASHBOARD, {
+            enrollmentId,
+            clientId,
+          })}
+          openInNew
+          sx={{ display: 'inline-flex' }}
```

Comment:
Interesting side note, `RouterLink` [internally](https://github.com/greenriver/hmis-frontend/blob/c37b83c10d89e6bf897e65d8241ad8c0a3eb1c44/src/components/elements/RouterLink.tsx#L27-L34) uses a `Stack` when `openInNew` is passed, with the following result:
- when `openInNew` is passed, by default `RouterLink` renders on its own line
- when `openInNew` is not passed, `RouterLink` renders inline with neighbors

That's why I added `sx={{ display: 'inline-flex' }}` here, but I also considered adding this sx directly on the `Stack` within `RouterLink`, to make that component's behavior more predictable. That would require a quick qa of all instances of `RouterLink` where `openInNew` is true, to make sure it doesn't change the ui in any way that we don't want.

Reply:
@martha thanks for flagging that. I had to look up the difference between `flex` and `inline-flex`. I think it makes sense that the RouterLink should use inline-flex by default when rendering the link next to an icon. Can you move this over to RouterLink? Thank you!

Reply:
As a quick aside, I noticed [ExternalLink](https://github.com/greenriver/hmis-frontend/blob/c37b83c10d89e6bf897e65d8241ad8c0a3eb1c44/src/components/elements/ExternalLink.tsx#L5) is a bit similar to RouterLink but it doesn't even use a stack, so it wouldn't have this problem. Should we just use that approach here?

Reply:
Belated follow-up:
- [x] ProjectOutgoingReferralPostingsTable.tsx - not impacted, because its neighbor `HmisEnum` renders a `<p>` internally which renders on its own line